usaon_benefit_tool/templates/macros/javascript_highcharts.j2

```diff
@@ -1,11 +1,21 @@
-{% macro add_javascript__highcharts(sankey_series) -%}
+{% macro add_javascript__highcharts(sankey_series,stitle) -%}
+
   <script>
+
+    Highcharts.getOptions().exporting.buttons.contextButton.symbol = 'download';
```

Comment:
For me, this made the menu look like a filled circle instead of the expected "hamburger" menu icon. I don't see "download" in the list of available options here: https://api.highcharts.com/highcharts/exporting.buttons.contextButton.symbol so I removed this line.

![image](https://github.com/nsidc/usaon-benefit-tool/assets/3608264/4f7c434f-588d-482d-aa7a-99b258e39c88)

![image](https://github.com/nsidc/usaon-benefit-tool/assets/3608264/13ec768f-0d06-403b-ae56-56266ab38566)